scripts/tailwind_checker.ts

```diff
@@ -0,0 +1,131 @@
+import { walk } from "https://deno.land/std@0.201.0/fs/mod.ts";
+
+const green = "\x1b[32m";
+const reset = "\x1b[0m";
+
+const folderPaths = ["./components", "./sections", "./islands"];
+const tailwindCssUrl =
```

Comment:
feeling that it should be get from somewhere related to the project's tailwind version or at least a way to specify as an env var, but you can keep it here

Reply:
About the env, ok about not doing this now.